common/core.css

```diff
@@ -36,7 +36,3 @@ body {
 	margin-right: auto;
 	margin-bottom: 0;
 }
-
-.editing {
```

Comment:
Instead of hiding edit features with CSS, I'm only adding them with JS if the person is admin (i.e. you)